src/bindings/python/python.cc

```diff
@@ -52,39 +51,70 @@ configure(const boost::optional<std::string>& config = boost::none) {
 void py_configure(const std::string& config_file) {
   configure(config_file);
 }
-
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(route_overloads, route, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(locate_overloads, locate, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(optimized_route_overloads, optimized_route, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(matrix_overloads, matrix, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(isochrone_overloads, isochrone, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(trace_route_overloads, trace_route, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(trace_attributes_overloads, trace_attributes, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(height_overloads, height, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(transit_available_overloads, transit_available, 1, 1);
-BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(expansion_overloads, expansion, 1, 1);
 } // namespace
 
-BOOST_PYTHON_MODULE(valhalla) {
+namespace py = pybind11;
+
+struct simplified_actor_t : public valhalla::tyr::actor_t {
+  simplified_actor_t(const boost::property_tree::ptree& config, bool auto_cleanup = false)
```

Comment:
may as well remove the `auto_cleanup` argument and pass true to the parent class here instead of setting to true below in the bindings